component/libfuse/libfuse.go

```diff
@@ -251,7 +252,7 @@ func (lf *Libfuse) Validate(opt *LibfuseOptions) error {
 //	Return failure if any config is not valid to exit the process
 func (lf *Libfuse) Configure(_ bool) error {
 	log.Trace("Libfuse::Configure : %s", lf.Name())
-
+	log.Info("Mount Command: %s", os.Args)
```

Comment:
This will print only once, lets add it to every log otherwise customer has to write scripts to figure out pid from here and then grep on logs to filter other logs from same mount.